app/src/pages/SingleContent.spec.ts

```diff
@@ -329,4 +332,21 @@ describe("SingleContent", () => {
             });
         });
     });
+
+    it("can bookmark a content", async () => {
```

Comment:
Can you also add a test to check if it can remove a bookmark?

The test should also check if the bookmark value is added to the bookmarks list in the global config

Reply:
This test should check if the userPreferences ref has been updated with the bookmark.

A test where a bookmarked document is "unbookmarked" should also be added (can be done in the same test)